Add Rigetti Provider Documentation (#563)
MicrosoftDocs/quantum-docs: articles/provider-support-rigetti.md

---

> author: kalzoo

Reviewer: ```suggestion
author: kalzoo
ms.author: brbenefield
```

---
author: kalzoo
ms.author: brbenefield
description: This document provides details on the support policy for the Rigetti provider in Azure Quantum
ms.date: 08/29/2022
ms.service: azure-quantum
ms.subservice: computing
ms.topic: reference
title: Support Policy for Rigetti in Azure Quantum
uid: microsoft.quantum.providers.rigetti.support
---

# Support Policy for Rigetti in Azure Quantum

> This article describes the Microsoft support policy that applies when you use the Rigetti provider in Azure Quantum. The article applies to any of the targets under this provider.

Reviewer: ```suggestion
This article describes the Microsoft support policy that applies when you use the Rigetti provider in Azure Quantum. The article applies to any of the [targets](xref:microsoft.quantum.providers.rigetti#targets) offered by the Rigetti provider.
```

> If you're using the Rigetti provider and experience unexpected issues, you can contact the Azure Support team for help 

Reviewer: ```suggestion
If you're using the Rigetti provider and experience unexpected issues, you can contact the Azure Support team for help.
```

## Support Points of Contact

In general, problems using the Rigetti provider on Microsoft Azure Quantum are best addressed with the Azure Quantum support team directly by [creating an Azure support case](/azure/azure-portal/supportability/how-to-create-azure-support-request). This team has all the necessary details to isolate the issue you're experiencing and direct the report to the right point of contact. The Azure support team also has the means to reach Rigetti support outside of business hours for the most urgent issues.

## Support Timeline

Rigetti will always attempt to resolve issues with the Microsoft Azure integration as quickly as practical. Some issues will take longer to address than others, but the tables below indicate the time you may expect for an initial response from Rigetti after an issue report.

| Severity   | Definition        | Response SLA        |
|------------|-------------------|---------------------|
| High | Failure in connectivity; complete performance degradation | 1 business day |
| Medium | Performance degradation | 2 business days |
| Low | Application-related concerns and questions | 5 business days |

> "Business days" are defined as Monday through Friday, not counting US holidays. Example: if a medium-severity incident is reported on a Tuesday, a response will be returned by the end of the business day on Thursday, if the intervening days are also business days.

Reviewer: ```suggestion
*Business days* are defined as Monday through Friday, not counting US holidays. For example, if a medium-severity incident is reported on a Tuesday, a response will be returned by the end of the business day on Thursday, if the intervening days are also business days.
```
